lua/lspconfig/server_configurations/vtsls.lua

```diff
@@ -0,0 +1,45 @@
+local util = require 'lspconfig.util'
+
+local cmd = { 'vtsls', '--stdio' }
+
+if vim.fn.has 'win32' == 1 then
+  cmd = { 'cmd.exe', '/C', unpack(cmd) }
+end
+
+return {
+  default_config = {
+    cmd = cmd,
+    filetypes = {
+      'javascript',
+      'javascriptreact',
+      'javascript.jsx',
+      'typescript',
+      'typescriptreact',
+      'typescript.tsx',
```

Comment:
I just noticed these composite filetypes (`javascript.jsx` and `typescript.tsx`). I just copied them from `tsserver` as this server is essentially a fork of that server. 

I don't believe these filetypes are valid as-is, per my understanding they should translate to:
- `typescript.tsx`: `typescript` & `tsx`
- `javascript.jsx`: `javascript` & `jsx`

@glepnir Do you happen to know if the above filetypes are indeed valid within lspconfig, or should I look into it further?